components/vf-sass-config/mixins/_typography.scss

```diff
@@ -1,53 +1,51 @@
 // Generate correct font information when included into an element
 // Recommended to use with typography and sizing maps
-// @include set-type(text-body--3, $global-font-family, $custom-margin-bottom: vf-spacing--lg)
+// @include set-type(text-body--3, $font-family, $custom-margin-bottom: vf-spacing--lg)
 
 // $custom-margin-bottom: variable, auto, disable
-@mixin set-type($font-size, $global-font-family: $global-font-family, $custom-margin-bottom: auto) {
-  @if $global-font-family != $vf-font-family--monospace {
```

Comment:
this was for that time when a team wanted to use a different typeface - but I guess that point is moot

Reply:
Briefly thought "should we add the font map as an optional parameter to the mixin?" But then I thought it might be overcomplicating things ... at some point when there's time (hah!) would be good to see just how much breaks when someone tries to use their own design tokens ... also get s back to the nightmare refactor when we have a `$colour-primary`.